libpod/options.go

```diff
@@ -636,3 +637,58 @@ func WithPodLabels(labels map[string]string) PodCreateOption {
 		return nil
 	}
 }
+
+// WithDNSSearch sets the additional search domains of a container
+func WithDNSSearch(searchDomains []string) CtrCreateOption {
+	return func(ctr *Container) error {
+		if ctr.valid {
+			return ErrCtrFinalized
+		}
+		if StringInSlice(".", searchDomains) && len(searchDomains) > 1 {
```

Comment:
Can you do this check in podman and make this a straight assignment? An empty []string can mean no search domains.

Reply:
fixed